docs/installation-aks.md

```diff
@@ -0,0 +1,407 @@
+
+You'll also want to ensure that you have at least three nodes in the cluster. You can use the `az aks scale` command to scale the number of worker nodes if needed.
+
+### New Cluster
+
+The following steps will walk you through creation of a new AKS cluster. For more information on cluster creation, please consult [Quickstart: Deploy an Azure Kubernetes Service Cluster](https://docs.microsoft.com/en-us/azure/aks/kubernetes-walkthrough) and [Create an Azure Kubernetes Service cluster](https://docs.microsoft.com/en-us/azure/aks/create-cluster).
+
+#### Create a Resource Group for AKS
+
+When you create an AKS cluster, you must provide a resource group. Create one with the az cli using the following command.
+
+```console
+az group create --name aks-group --location eastus
+```
+
+#### Create a Kubernetes cluster using AKS
+
+* As AKS is currently in preview, you will need to enable it in your subscription
+* AKS currently does _not_ support RBAC, so we will need to explicity disable that when we install service catalog.
+
+1. Enable AKS in your subscription, use the following command with the az cli:
+    ```console
+    az provider register -n Microsoft.ContainerService
+    ```
+
+You should also ensure that the `Microsoft.Compute` and `Microsoft.Network` providers are registered in your subscription. If you need to enable them:
+    ```console
+    az provider register -n Microsoft.Compute
+    az provider register -n Microsoft.Network
+    ```
+
+1. Create the AKS cluster!
+    ```console
+    az aks create --resource-group aks-group --name aks-cluster --generate-ssh-keys --kubernetes-version 1.9.6
+    ```
+
+    Note: Service Catalog may not work with Kubernetes versions less than 1.9.0. If you are attempting to use an older AKS cluster, you will need to upgrade. The earliest 1.9.x release available from AKS is 1.9.1, so you will need to upgrade to at least that version.
+
+1. Configure kubectl to use the new cluster
+    ```console
+    az aks get-credentials --resource-group aks-group --name aks-cluster
+    ```
+
+1. Verify your cluster is up and running
+    ```console
+    kubectl get nodes
+    ```
+
+---
+
+## Cluster Configuration
+
+Before you can install OSBA onto your cluster, you will first need to install Service Catalog. Service Catalog uses etcd for data persistence and it is *strongly* recommended that you create an etcd cluster with backup and recovery capabilities. We recommend using [etcd operator](https://github.com/coreos/etcd-operator) configured with backup and recovery via Azure Blob Storage for this purpose. The following section provides guidance on how to configure etcd operator on your cluster and then how to install Service Catalog.
+
+### Install Helm
+
+We will use Helm to install etcd Operator, Service Catalog and OSBA. In order to install Helm in your cluster, use the `helm` CLI:
+
+```console
+helm init
```

Comment:
Is this the most safe way to run helm in prod? I'm wondering if since we are kind of saying this is prod safe way to install on aks that we may want to include any other security best practices in this command if necessary.

Reply:
I believe the more proper way is to create a service account and create the RBAC policies. We do this in the quickstarts now, so i have updated this. I'll ping Matt F and see if he has any more feedback.